logs_http.module

```diff
@@ -136,7 +136,8 @@ function logs_http_register_event(array $log_entry) {
     'uid' => $log_entry['uid'],
     'link' => strip_tags($log_entry['link']),
     'message' => empty($log_entry['variables']) ? $log_entry['message'] : strtr($log_entry['message'], $log_entry['variables']),
-    'severity' => $log_entry['severity'],
+    'severity' => watchdog_severity_levels()[$log_entry['severity']],
```

Comment:
Since travis is set to check `php 5.3` this convention is not valid. We should assign the `watchdog_severity_levels()` to a variable and then use it with the `$log_entry['severity']` index.